src/client/Microsoft.Identity.Client.KeyAttestation/Microsoft.Identity.Client.KeyAttestation.csproj

```diff
@@ -19,20 +19,14 @@
     <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
     <Version>$(MicrosoftIdentityClientVersion)-preview</Version>
     <!-- Copyright needs to be in the form of © not (c) to be compliant -->
-    <Title>MSAL.NET extension for managed identity proof-of-possession flows</Title>
+    <Title>MSAL.NET extension for KeyGuard attestation support</Title>
```

Comment:
the feature GA'd under this name - https://learn.microsoft.com/en-us/windows/security/identity-protection/credential-guard/

Reply:
All comments should refer to this as Credential Guard.